client/src/app/site/common/components/search/search.component.html

```diff
@@ -1,58 +1,72 @@
 <os-head-bar [nav]="false" [goBack]="true">
     <!-- Title -->
     <div class="title-slot"><h2 translate>Search results</h2></div>
+
     <!-- Menu -->
     <div class="menu-slot">
         <button type="button" mat-icon-button [matMenuTriggerFor]="menu"><mat-icon>more_vert</mat-icon></button>
     </div>
 </os-head-bar>
 
 <!-- search-field -->
-<div class="search-field">
-    <form [formGroup]="quickSearchform">
-        <mat-form-field>
-            <input matInput osAutofocus formControlName="query" (keyup)="quickSearch()" />
+<div class="search-container">
+    <div class="search-field">
+        <mat-form-field appearance="outline" class="search-component">
+            <mat-label>{{ 'Search' | translate }}</mat-label>
+            <input #searchBox matInput osAutofocus (input)="quickSearch(searchBox.value)" />
+            <button mat-icon-button matPrefix (click)="quickSearch('')">
+                <mat-icon>close</mat-icon>
+            </button>
             <mat-icon matSuffix>search</mat-icon>
         </mat-form-field>
-    </form>
-</div>
 
-<mat-menu #menu="matMenu">
-    <button mat-menu-item *ngFor="let registeredModel of registeredModels" (click)="toggleModel(registeredModel)">
-        <mat-checkbox [checked]="registeredModel.enabled" (click)="$event.preventDefault()">
-            <span>{{ registeredModel.verboseNamePlural | translate }}</span>
-        </mat-checkbox>
-    </button>
-</mat-menu>
+        <mat-form-field class="search-sort">
+            <mat-label>{{ 'Sort' | translate }}</mat-label>
+            <mat-select [(value)]="sortingProperty" (selectionChange)="search()">
+                <mat-option value="title">{{ 'Title' | translate }} </mat-option>
+                <mat-option value="id">ID</mat-option>
+            </mat-select>
+        </mat-form-field>
+    </div>
 
-<div class="noSearchResults" *ngIf="searchResultCount === 0">
-    <span translate>No search result found for</span> "{{ query }}"
-</div>
+    <mat-menu #menu="matMenu">
+        <button mat-menu-item *ngFor="let registeredModel of registeredModels" (click)="toggleModel(registeredModel)">
+            <mat-checkbox [checked]="registeredModel.enabled" (click)="$event.preventDefault()">
+                <span>{{ registeredModel.verboseNamePlural | translate }}</span>
+            </mat-checkbox>
+        </button>
+    </mat-menu>
 
-<ng-container *ngIf="searchResultCount > 0">
-    <h3>
-        {{ searchResultCount }}
-        <span *ngIf="searchResultCount === 1" translate>result</span>
-        <span *ngIf="searchResultCount > 1" translate>results</span>
-    </h3>
+    <div *ngIf="searchResults.length > 0">
```

Comment:
according to out mock-ups, the whole thing could be a `mat-card`, or something else with a plain white background.
The first entry could be a `mat-divider`, to separate the search from the results.